man/roxygen/meta.R

```diff
@@ -0,0 +1,17 @@
+list(
+  rd_family_title = list(
+    adjacency = "Adjacency matrices",
+    attributes = "Vertex, edge and graph attributes",
+    bipartite = "Bipartite graphs",
+    centrality = "Centrality measures",
+    coloring = "Graph coloring",
+    community = "Community detection",
+    components = "Connected components",
+    epi = "Processes on graphs",
```

Comment:
or should it remain "epi"?

Reply:
I would rename it `processes`, as non epidemiological functionality may go into this section later.